input/includes/scope-note.md

```diff
@@ -1,61 +1,38 @@
 <div class="request-for-feedback" markdown="1">
 
 #### AU Core R1 Scope 
```

Comment:
A request for feedback no longer makes sense here. 
1. Turn it into Note to Contributors that is the difference between last snapshot and this one.
1. Remove all requests for feedback from this page.

Reply:
Updated note & removed requests for feedback

Reply:
1. Note should be difference from last snapshot; it isn't a note on the scope changes made by this PR for March for Ballot scope. Change the content and tone to be that diff.
2. confirmed.

Reply:
Change made to be about the difference between the snapshot and this one (not yet pushed).

Reply:
that will do we can fix after merge.